packages/main/src/MultiInput.js

```diff
@@ -164,11 +172,59 @@ class MultiInput extends Input {
 	_onkeydown(event) {
 		super._onkeydown(event);
 
+		if (isLeft(event)) {
+			this._skipOpenSuggestions = true;
+
+			return this._handleLeft(event);
+		}
+
+		this._skipOpenSuggestions = false;
```

Comment:
Initialize the flag in the constructor and explain what is needed for in a comment